sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 5.7.0-beta.2 (Unreleased)
+## 5.7.0-beta.2 (2022-02-08)
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
+- `createEventDataAdapter` is added which specifies how to convert an `EventData` to a simple message with `body` and `contentType` fields. This adapter can be used with the Avro encoder in `@azure/schema-registry-avro` starting from version 1.0.0-beta.6 to create `EventData` messages with Avro-encoded bodies.
 
-### Other Changes
+- Add idempotent publishing support to `EventHubBufferedProducerClient`.  The functionality can be enabled by setting the `enableIdempotentPartition` client option to `true`.  [PR #20156](https://github.com/Azure/azure-sdk-for-js/pull/20156)
```

Comment:
Can we add a single line description of what idempotent publishing offers to the user? Otherwise I am worried people might expect that they can send the same message multiple times and it would result only in 1 message in the service.

Reply:
I copied some description from .NET.